plugins/modules/dcnm_vpc_pair.py

```diff
@@ -0,0 +1,1315 @@
+
+          KEEP_ALIVE_VRF:
+            description:
+              - Name of the VRF used for keep-alive messages.
+            type: str
+            required: true
+
+          PC_MODE:
+            description:
+              - Port channel mode.
+            type: str
+            choices: ['on', 'active', 'passive']
+            default: active
+
+          PEER1_DOMAIN_CONF:
+            description:
+              Additional CLI for PEER1 vPC Domain.
+            type: str
+            default: ""
+
+          PEER1_KEEP_ALIVE_LOCAL_IP:
+            description:
+              - IP address of a L3 interface in non-default VRF on PEER1.
+            type: str
+            required: true
+
+          PEER1_MEMBER_INTERFACES:
+            description:
+              - A list of member interfaces for PEER1.
+            type: list
+            elements: str
+            default: []
+
+          PEER1_PCID:
+            description:
+              - PEER1 peerlink port-channel number.
+              - Minimum value is 1 and Maximum value is 4096.
+            type: int
+            default: 1
+
+          PEER1_PO_CONF:
+            description:
+              - Additional CLI for PEER1 vPC peerlink port-channel.
+            type: str
+            default: ""
+
+          PEER1_PO_DESC:
+            description:
+              - Description for the PEER1 port-channel.
+              - Minimum length is 1 and Maximum length is 254.
+            type: str
+            default: ""
+
+          PEER2_DOMAIN_CONF:
+            description:
+              Additional CLI for PEER2 vPC Domain.
+            type: str
+            default: ""
+
+          PEER2_KEEP_ALIVE_LOCAL_IP:
+            description:
+              - IP address of a L3 interface in non-default VRF on PEER2.
+            type: str
+            required: true
+
+          PEER2_MEMBER_INTERFACES:
+            description:
+              - A list of member interfaces for PEER2.
+            type: list
+            elements: str
+            default: []
+
+          PEER2_PCID:
+            description:
+              - PEER2 peerlink port-channel number.
+              - Minimum value is 1 and Maximum value is 4096.
+            type: int
+            default: 1
+
+          PEER2_PO_CONF:
+            description:
+              - Additional CLI for PEER2 vPC peerlink port-channel.
+            type: str
+            default: ""
+
+          PEER2_PO_DESC:
+            description:
+              - Description for the PEER2 port-channel.
+              - Minimum length is 1 and Maximum length is 254.
+            type: str
+            default: ""
+
+  templates:
+    description:
+      - List of templates to be fetched.
+      - This is required only if the 'state' is 'fetch'. In this case the list should contain the template names whose details.
+        are to be fetched.
+    type: list
+    elements: str
+    default: []
+"""
+
+EXAMPLES = """
+
+# States:
```

Comment:
The description for "state" in the DOCUMENTATION section says that "fetch" state is supported.  But "fetch" is not described below.  Would be good to add a description for Fetch, as has been done for Merged, Replaced, etc.